tidb-cloud/terraform/terraform-overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: TiDB Cloud Terraform Provider Overview
+summary: Create, manage, and update your TiDB Cloud resources through Terraform
+---
+
+# TiDB Cloud Terraform Provider Overview
+
+As a fully managed service of TiDB, TiDB Cloud can automate database maintenance operations. With the [TiDB Cloud API](https://docs.pingcap.com/tidbcloud/api/v1beta), you can programmatically handle more operations such as deployment, scaling, and restore.
+
+If you are looking for a simplified way to automate resource provisioning and your infrastructure workflow. You can try out our [TiDB Cloud Terraform provider](https://registry.terraform.io/providers/tidbcloud/tidbcloud), which provides you with the following capabilities:
+
+- Get your project information.
+- Get cluster specification information, like supported cloud_provider, region, node size and so on.
+- Manage your TiDB cluster, including creating, scaling, pausing and resuming a cluster.
+- Create and delete the backup for your cluster.
+- Create the restore task for your cluster.
+
+## Requirements
+
+- [TiDB Cloud](https://docs.pingcap.com/tidbcloud/release-notes): compatible with TiDB Cloud that is released at September 27, 2022
```

Comment:
```suggestion
- [TiDB Cloud](https://docs.pingcap.com/tidbcloud/release-notes): compatible with TiDB Cloud that is released since September 27, 2022
```